README.md

```diff
@@ -164,6 +164,51 @@ To capture the exported code from the container and remove the Docker objects, e
 A [volume](https://docs.docker.com/storage/volumes/) could also be mounted to the container.
 
 
+### Tagging an image to upload to Docker Hub
+First, you need to login to Docker Hub using the command `docker login`.
+
+MAKE SURE THAT WHAT YOU PUSH IS OPEN SOURCE CODE. Docker Hub is a public
+resource. There are enterprise versions of Docker Hub; we are NOT providing
+intructions for that here.
+
+Then you need to tag your image. First find the image ID using `docker images`. For example,
+
+    REPOSITORY           TAG                 IMAGE ID            CREATED             SIZE
+    wv                   latest              4d088aa275ff        13 hours ago        6.06GB
+    vehu                 latest              ed6175534a1c        20 hours ago        4.29GB
+    vxvista              latest              3866735fcfc0        21 hours ago        2.96GB
+    cachevista           latest              071386cb0e80        46 hours ago        15.4GB
+    centos               latest              75835a67d134        11 days ago         200MB
+    osehra/osehravista   latest              8d58b9b985d7        5 weeks ago         4.63GB
+    hello-world          latest              e38bc07ac18e        6 months ago        1.85kB
+
+So, if we want to push `wv` up, then we need to use the image ID `4d088aa275ff`
```

Comment:
you can also shortcut this by doing docker build -t boo/wv:latest -t boo/wv:v3, etc.